flexbridge

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Run command in environment
+
+set -e -o pipefail
+
+scriptdir="$(dirname "$0")"
+prefix=$(cd "$scriptdir/../.."; /bin/pwd)
+
+cd "$(dirname "$0")"
+. environ
+
+(
+	XDG_DATA_HOME=${XDG_DATA_HOME:-${HOME}/.local/share}
+	FB_SHARE="${XDG_DATA_HOME}/SIL/FlexBridge"
+
+	# Keep localizations files updated.
+	# Initialize on fresh install
+	if [ ! -f "${FB_SHARE}" ]; then
+		mkdir -p "${FB_SHARE}"
+	fi
+	
+	# update new localizations
+	if [ "${prefix}" == "/usr" ];then
+		cp -a "/var/lib/flexbridge/localizations" "${FB_SHARE}"
+	else
+		# For developer build. This is still not working correctly.
+		cp -a "${prefix}/DistFiles/localizations" "${FB_SHARE}"
+	fi
+		
+)
+
+cd - >/dev/null
+"$@"
```

Comment:
Right, so I think line 34 will need to be set to something like

    "$(dirname "$0")/"FlexBridge.exe "$@"